### PR TITLE
3/deprecated docs

### PR DESCRIPTION
**Pull request by abea**
Clears deprecated documentation and features no longer relevant to 3.x

### CODE REVIEW

File: modules/@apostrophecms/admin-bar/index.js [hunk 1]
```diff
@@ -3,22 +3,6 @@
 // module. Buttons can also be grouped into dropdown menus and restricted to those with
 // particular permissions. [@apostrophecms/piece-type](../@apostrophecms/piece-type/index.html) automatically
 // takes advantage of this module.
-//
-// The admin bar slides out on all pages by default. It's possible to modify this behavior
```
Comment (abea) [inline modules/@apostrophecms/admin-bar/index.js:7]: Our admin bar no longer slides out.

File: modules/@apostrophecms/page/index.js [hunk 2]
```diff
@@ -13,36 +13,6 @@ module.exports = {
         label: 'Home'
       }
     ],
-    contextMenu: [
```
Comment (abea) [inline modules/@apostrophecms/page/index.js:16]: There is no context menu anymore.

File: modules/@apostrophecms/page/index.js [hunk 3]
```diff
@@ -1494,46 +1464,9 @@ database.`);
         const args = {
           edit: providePage ? req.data.bestPage._edit : null,
           slug: providePage ? req.data.bestPage.slug : null,
-          page: providePage ? req.data.bestPage : null,
-          contextMenu: req.contextMenu,
-          publishMenu: req.publishMenu
+          page: providePage ? req.data.bestPage : null
         };
-        if (args.page && args.edit) {
```
Comment (abea) [inline modules/@apostrophecms/page/index.js:1501]: all context menu  and publish menu stuff

File: modules/@apostrophecms/piece-page-type/index.js [hunk 4]
```diff
@@ -37,37 +37,6 @@ module.exports = {
 
     self.piecesFilters = self.options.piecesFilters || [];
 
-    self.contextMenu = options.contextMenu || [
```
Comment (abea) [inline modules/@apostrophecms/piece-page-type/index.js:40]: more context menu

File: modules/@apostrophecms/piece-type/index.js [hunk 5]
```diff
@@ -4,7 +4,6 @@ module.exports = {
   extend: '@apostrophecms/doc-type',
   cascades: [ 'filters', 'columns', 'batchOperations' ],
   options: {
-    manageViews: [ 'list' ],
```
Comment (abea) [inline modules/@apostrophecms/piece-type/index.js:7]: No longer using `manageViews`

File: modules/@apostrophecms/piece-type/index.js [hunk 6]
```diff
@@ -745,6 +739,7 @@ module.exports = {
           {
             type: 'major',
             label: 'New ' + self.options.label,
+            // TODO: fully deprecate `insertViaUpload`
```
Comment (abea) [inline modules/@apostrophecms/piece-type/index.js:742]: Our image uploader is custom built so doesn't need this anymore. Our file uploader will likely get its own treatment as well, I'd imagine.

File: modules/@apostrophecms/piece-type/index.js [hunk 7]
```diff
@@ -839,7 +834,6 @@ module.exports = {
         // Options specific to pieces and their manage modal
         browserOptions.filters = self.filters;
         browserOptions.columns = self.columns;
-        browserOptions.contextual = self.contextual;
```
Comment (abea) [inline modules/@apostrophecms/piece-type/index.js:842]: this stuff seemed to be driving the context menu options.

Comment (boutell) [reply modules/@apostrophecms/piece-type/index.js:842]: I would mark this one with a TODO instead as the UI does need to know if we decide to keep the idea of a configuration flag for whether to redirect on create.

**Review by boutell — CHANGES_REQUESTED** [review]
This is extremely 💯, thank you for doing it. One quibble.

**Review by boutell — APPROVED** [review]
So you want to kill the contextual flag for now but we're keeping an open mind as to there being some kind of mechanism in future. OK.

**Pull request merged**